### PR TITLE
Update Search pages

### PR DESCRIPTION
**Pull request by ouafaaetta**
Update Search page with a new Search disclaimer message in WET-335

### CODE REVIEW

File: templates/search/results-advanced-en.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
 			<label for="advseacon4">none of these words:</label>
 			<input name="noneq" class="form-control" id="advseacon4" maxlength="100" data-fusion-query="safe" aria-describedby="gc-pi" >
 		</div>
-		<p id="gc-pi" class="mrgn-tp-md">Please <strong>do not include sensitive personal information</strong> in the search box, such as your social insurance number, personal finance data and medical or work history.</p>
+		<p id="gc-pi" class="mrgn-tp-md">Don’t include personal information (telephone, email, SIN, financial, medical, or work details).</p>
```
Comment (GormFrank) [inline templates/search/results-advanced-en.html:32]:
```suggestion
		<p id="gc-pi" class="mrgn-tp-md">Don't include personal information (telephone, email, SIN, financial, medical, or work details).</p>
```

File: templates/search/results-contextual-en.html [hunk 2]
```diff
@@ -29,7 +29,7 @@
 			</div>
 			<div class="clearfix"></div>
 		</div>
-		<p id="gc-pi" class="mrgn-tp-md">Please <strong>do not include sensitive personal information</strong> in the search box, such as your social insurance number, personal finance data and medical or work history.</p>
+		<p id="gc-pi" class="mrgn-tp-md">Don’t include personal information (telephone, email, SIN, financial, medical, or work details).</p>
```
Comment (GormFrank) [inline templates/search/results-contextual-en.html:32]:
```suggestion
		<p id="gc-pi" class="mrgn-tp-md">Don't include personal information (telephone, email, SIN, financial, medical, or work details).</p>
```

**Review by GormFrank — CHANGES_REQUESTED** [review]
Svp utiliser l'apostrophe uniformément : `'` au lieu de `’`

**Review by ouafaaetta — COMMENTED** [review]
l'apostrophe ' au lieu de ’ est utilisé dans tous les fichiers uniformément

**Review by GormFrank — CHANGES_REQUESTED** [review]

**Review by GormFrank — CHANGES_REQUESTED** [review]
@ouafaaetta Tel que discuté; même corrections nécessaires en français.

Puis corriger les erreurs identifiés par le script "Continuous integration".

Merci :)

**Review by GormFrank — CHANGES_REQUESTED** [review]
@ouafaaetta Dernier petit changement, et aussi régler les problèmes identifiés par le **Build** (continuous integration) svp.

**Review by duboisp — COMMENTED** [review]
pre-approved upon successful testing and review

**Review by GormFrank — APPROVED** [review]

**Review by duboisp — APPROVED** [review]
Tested and reviewed and this PR is ready to be merged. 

This is a major change for the template but it is a "moderated used" template according to our versioning API. We will need to specify in the release note that a communication about that would be sent in June 2023 so we will be able to mark it as a minor change for the GCWeb product.

@ouafaaetta There is only the following two content update which you can quickly do later this week:
* [x] add in the `pages.examples` grouping "results-advanced-en.html" and "result-advanced-fr.html" in the index.json-ld file
* [ ] add in the `pages.docs` grouping  "api-fr.html" in the index.json-ld

**Pull request merged**